action.yml

```diff
@@ -0,0 +1,28 @@
+name: "vulnbot-action"
+description: ""
+inputs:
+  config:
+    default: config.toml
+    description: "Path that specifies the ecosystems for which the VulnBot will be reporting vulnerabilities."
+    required: false
+  slack_auth_token:
+    description: "Represents the Slack authentication token that allows the VulnBot to access and post messages to a Slack channel."
+    required: false
+  github_token:
+    description: "Access token that allows the VulnBot to interact with the GitHub API."
+    required: false
+  github_org:
+    description: "Refers to the GitHub organization that the repository belongs to."
+    required: false
+runs:
+  using: docker
+  image: Dockerfile
+  env:
+    SLACK_AUTH_TOKEN: "${{ inputs.slack_auth_token }}"
+    GITHUB_TOKEN: "${{ inputs.github_token }}"
+    GITHUB_ORG: "${{ inputs.github_org }}"
+  args:
+    - "--config=${{ inputs.config }}"
```

Comment:
I think we're going to need to mount the config file as a volume in the Docker container, otherwise it won't be able to read the file. The example I give in the `vulnbot` README is:
```
docker run --env-file .env -v ./config.toml:/app/config.toml vulnbot
```

Reply:
So we really don't need to do that, since we're letting Cobra pull in the config file from the flag specified by the user. This is the actual code we have today:

`configFilePath := getString(cmd.Flags(), "config")`

Reply:
Right, but Cobra will be trying to pull the file in from the Docker filesystem, and it won't exist there unless we explicit copy or mount it in there

Reply:
That makes sense, so this would happen If we don't pass the config file path through the flag, right?